docs/reference/release-notes/8.0.0-beta1.asciidoc

```diff
@@ -13,9 +13,9 @@ Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
 === Known issues
 
 * If you're using {ml}, it's not safe to perform a rolling upgrade from `7.16.x`
-  to `8.0.0-beta1`. Doing so might lead to cluster state propagation issues in the mixed 
-  version cluster. We recommend waiting for `8.0.0-rc1` before testing a 
-  rolling upgrade from `7.16.x`. Do not attempt to upgrade any 
+  to `8.0.0-beta1`. Doing so might lead to cluster state propagation issues in the mixed
+  version cluster. We recommend waiting for `8.0.0-rc1` before testing a
+  rolling upgrade from `7.16.x`. Do not attempt to upgrade any
```

Comment:
Super nit: Did you mean to alter these lines? If not can you revert them for a cleaner blame?

Reply:
The short answer is that I didn't mean to and will revert. The long answer begins with a `git rebase`...